docs_devel/docs/93.BuildingInstallerPackage.md

```diff
@@ -129,75 +119,78 @@ OmegaT project prepares tasks to sign to packages for macOS.
 
 - linux64DistTarBz
 - linuxArm64DistTarBz
 
-These tasks build `tar.bz2` distribution package with JRE.
+These tasks build a `tar.bz2` distribution package that includes a JRE.
 
-## Java Runtime to bundle with
+## Bundled Java Runtime Environments
 
-You need to put Java Runtime package in a directory `assetDir` property specified.
-It is a parent directory of OmegaT project source in default.
+You need to put a Java Runtime package in the directory specified by the `assetDir` property. By default, it is the parent directory of the OmegaT project source directory.
 
-Following packages are recognized as a windows JRE.
+The following packages are recognized as windows JREs:
 
 - OpenJDK17U-jre_x86-32_windows_hotspot_17.0.8.1_1.zip
 - OpenJDK17U-jre_x64_windows_hotspot_17.0.8.1_1.zip
 
-Following packages are recognized as a macOS JRE.
+The following packages are recognized as macOS JREs:
 
 - OpenJDK17U-jre_aarch64_mac_hotspot_17.0.8.1_1.tar.gz
 - OpenJDK17U-jre_x64_mac_hotspot_17.0.8.1_1.tar.gz
 
-Following packages are recognized as a Linux JRE.
+The following packages are recognized as Linux JREs:
 
 - OpenJDK17U-jre_aarch64_linux_hotspot_17.0.8.1_1.tar.gz
 - OpenJDK17U-jre_x64_linux_hotspot_17.0.8.1_1.tar.gz
 
 
 ## Building Linux native packages
 
-OmegaT has tasks to generate Linux native packages in "DEB" and "RPM".
-We use `jpackage` packaging tool which is added in a Java Development Kit.
-It has been introduced from Java 14 and OmegaT project requires JDK 17.
+OmegaT has tasks to generate Linux native packages in "DEB" and "RPM" archive formats.
+We use the `jpackage` packaging tool which is provided by Java Development Kits.
+The tool was introduced in Java 14, which means the OmegaT project requires JDK 17
+to build packages.
 
 The packaging tool `jpackage` generates installable packages for Java applications.
-You can provide platform-specific packages for Linux, macOS and Windows to users
-with a familiar way to install and launch your applications.
+You can provide platform-specific packages for Linux, macOS or Windows that can be
+installed and launched in ways familiar to user.
 
-`jpackage` bundles java runtime which is used to run Gradle or is used to compile a project
-that has `jpackage` utility. Because OmegaT project uses Java 11 to compile, you need to
-run Gradle with JDK17.
+`jpackage` bundles the java runtime used to run Gradle, but can also be run with the runtime
+used to compile the project that relies on the `jpackage` utility. Fpr OmegaT, you need to
+run Gradle with JDK 17 to create packages, because the application itself is compiled with
+Java 11, which is not compatible with `jpackage`.
 
-OmegaT use `gradle-jpackage-plugin` that integrate `jpackage` command line utility
-which is bundled in JDK with Gradle build utility.
+OmegaT uses the `gradle-jpackage-plugin`, which integrates the `jpackage` command line utility
+bundled in the JDK into the Gradle build tool.
+-> I do not understand this sentence
```

Comment:
> -> I do not understand this sentence
I just took a stab at rewriting it.
@miurahr: Is this what you meant?

Reply:
Yes

Reply:
Thank you, @miurahr.

@brandelune: If this change works for you, I think we're ready to delete your "->" comment line and merge this set of changes.